### PR TITLE
Refactored CSS module

### PR DESCRIPTION
**Pull request by Robel-003**
Can't seem to find the right resolve, but this pr is for issue 5(UA/UI landing page), in sprint 1

### CODE REVIEW

**Review by taylorpapke — DISMISSED** [review]
Looks good to me!

File: app/page.tsx [hunk 1]
```diff
@@ -1,44 +1,45 @@
 import React from 'react';
+import styles from './login.module.css';
```
Comment (tinpham5614) [inline app/page.tsx:2]: I received an error at this line. I think you meant `'import styles from './login-page.module.css'`

Comment (Robel-003) [reply app/page.tsx:2]: I forgot to update the imports. I'll fix those imports and create a new pr.

Comment (tinpham5614) [reply app/page.tsx]: No more errors after updating, the web is working now.

**Review by taylorpapke — APPROVED** [review]
LGTM

**Review by Seiyaroo — APPROVED** [review]
LGTM

**Pull request merged**